src/mobile-ui-react/ResizablePanel.tsx

```diff
@@ -414,6 +419,49 @@ class TouchCaptor extends React.PureComponent<TouchCaptorProps> {
   };
 }
 
+// @todo Use the functional version of TouchCaptor below after sufficient testing.
+// // 2024-06-24: Converted to a function due to compile errors that I couldn't understand after
```

Comment:
Remove mention of compile error since that is now resolved

Reply:
Fixed.